### ISSUE TITLE
Can't untag numeric tag name (str('1')) from an event

### ISSUE THREAD

**Issue by aaronkaplan** [MISP/PyMISP#1272]
If you tag an event with a numeric tag name (let's say the string '1'), then it's impossible to `misp.untag(...)` it again from an event.

See    line:
```python
     response = self._prepare_request('POST', 'tags/removeTagFromObject', data=to_post)
```

It's probably connected to the way the `data` is processed in MISP and not really a PyMISP issue. Maybe there is a work-around in PyMISP (?)

And indeed, in MISP the bug is here:
https://github.com/MISP/MISP/blob/c3221af12e664daf501afeef0a658fb11e018f58/app/Controller/TagsController.php#L876

**Comment by Rafiot** [comment]
I don't think there is a way to escape the string so the tag is removed instead of being converted to an int. @iglocska @mokaddem @righel if you have an idea?

**Comment by aaronkaplan** [comment]
Yeah, I was afraid of that. The web serivce just gets data and decides if it should be interpreted as int() first or not...
